errors.go

```diff
@@ -4,6 +4,7 @@ package iso8583
 // connection failed to unpack message
 type UnpackError struct {
 	Err        error
+	Field      string
 	RawMessage []byte
 }
 
```

Comment:
Is it worth updating `Error()` method to also interpolate the `Field`? But don't know whether this might be seen as a breaking change